src/Controls/src/Core/Switch/Switch.cs

```diff
@@ -21,7 +21,7 @@ public partial class Switch : View, IElementConfiguration<Switch>, ISwitch
 			((Switch)bindable).Toggled?.Invoke(bindable, new ToggledEventArgs((bool)newValue));
 			((Switch)bindable).ChangeVisualState();
 			((IView)bindable)?.Handler?.UpdateValue(nameof(ISwitch.TrackColor));
-
+			((IView)bindable)?.Handler?.UpdateValue(nameof(ISwitch.ThumbColor));
```

Comment:
Maybe with the new work @StephaneDelcroix did with style specificities, this is no longer needed and the style should swap?